babel/numbers.py

```diff
@@ -561,6 +563,18 @@ def __init__(self, pattern, prefix, suffix, grouping,
         else:
             self.scale = 0
 
+    def __copy__(self):
```

Comment:
Hmm, so is a concrete implementation of `__copy__` here actually even necessary?

I mean, wouldn't the builtin implementation of copy just basically do the same -- create a new instance and shallow-copy the instance properties?